ddtrace/appsec/_iast/taint_sinks/stacktrace_leak.py

```diff
@@ -0,0 +1,102 @@
+import os
+import re
+
+from ..._constants import IAST_SPAN_TAGS
+from .. import oce
+from .._iast_request_context import set_iast_stacktrace_reported
+from .._metrics import _set_metric_iast_executed_sink
+from .._metrics import increment_iast_span_metric
+from .._taint_tracking._errors import iast_taint_log_error
+from ..constants import HTML_TAGS_REMOVE
+from ..constants import STACKTRACE_EXCEPTION_REGEX
+from ..constants import STACKTRACE_FILE_LINE
+from ..constants import VULN_STACKTRACE_LEAK
+from ..taint_sinks._base import VulnerabilityBase
+
+
+@oce.register
+class StacktraceLeak(VulnerabilityBase):
+    vulnerability_type = VULN_STACKTRACE_LEAK
+    skip_location = True
+
+
+def asm_report_stacktrace_leak_from_django_debug_page(exc_name, module):
+    increment_iast_span_metric(IAST_SPAN_TAGS.TELEMETRY_EXECUTED_SINK, StacktraceLeak.vulnerability_type)
+    _set_metric_iast_executed_sink(StacktraceLeak.vulnerability_type)
```

Comment:
It's redundant with line 97

Reply:
Nope, different functions called at different times (django debug page vs other cases)